### PR TITLE
Bump openexr to 2.2.0 (update from upstream)

### PR DESCRIPTION
**Pull request by alarcher**
After #3116

### CODE REVIEW

File: components/library/openexr/openexr.p5m [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+#
```
Comment (pyhalov) [inline components/library/openexr/openexr.p5m]: Hm.. Is it auto-generated or borrowed from upstream?

**Comment by pyhalov** [comment]
Please, also remove openexr from components/components.ignore.

**Comment by pyhalov** [comment]
Build fails with:
/export/home/alp/srcs/tests/oi-userland/components/library/openexr/openexr-2.2.0/IlmImf/ImfSystemSpecific.cpp: In function ‘void Imf_2_2::{anonymous}::cpuid(int, int&, int&, int&, int&)’:
/export/home/alp/srcs/tests/oi-userland/components/library/openexr/openexr-2.2.0/IlmImf/ImfSystemSpecific.cpp:51:29: error: inconsistent operand constraints in an ‘asm’
             : /* Clobber */);

Quick googling finds this:  https://forums.freebsd.org/threads/47674/

**Comment by alarcher** [comment]
Nah... I just forgot to commit the patch...

**Comment by pyhalov** [comment]
ping

**Comment by alarcher** [comment]
One test crashes (opening a non-existing file) in 64-bit ...

**Comment by alarcher** [comment]
There is no additional patch known to fix this, I only found a bug report from several years ago with the same issue.

**Comment by pyhalov** [comment]
Have you looked at patch mentioned at https://forums.freebsd.org/threads/47674/ ?

**Comment by alarcher** [comment]
I already have it in my tree, it is another issue.

**Comment by alarcher** [comment]
Fixed: the issue was with libtool in ilmbase.
Do you want to wait until these are solved.? https://github.com/openexr/openexr/issues/232

**Comment by pyhalov** [comment]
Perhaps, just integrate patch from https://github.com/openexr/openexr/pull/233 ?

**Comment by alarcher** [comment]
Updated with the patch, omitted the tests as it relies on binary files.

**Pull request merged**